src/API/Plugins.php

```diff
@@ -431,6 +445,34 @@ public function activate_plugins( $request ) {
 		) );
 	}
 
+	/**
+	 * Return recommend payment plugins.
```

Comment:
should be "recommended"?

Reply:
It's not the first time I spelled it like that haha